src/main/java/com/jmb/springfactory/dao/productionorder/ProductionOrderMySQLImpl.java

```diff
@@ -1,16 +1,24 @@
 package com.jmb.springfactory.dao.productionorder;
 
+import java.util.Arrays;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)